BUILD

```diff
@@ -0,0 +1,60 @@
+# Description:
+# A Motion Animator creates performant, interruptible animations from motion specs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+load("@bazel_ios_warnings//:strict_warnings_objc_library.bzl", "strict_warnings_objc_library")
+
+strict_warnings_objc_library(
+    name = "MotionAnimator",
+    srcs = glob([
+        "src/*.m",
+        "src/private/*.m",
+    ]),
+    hdrs = glob([
+        "src/*.h",
+        "src/private/*.h",
+    ]),
+    deps = [
+      "@motion_interchange_objc//:MotionInterchange"
+    ],
+    enable_modules = 1,
+    includes = ["src"],
+    visibility = ["//visibility:public"],
+)
+
+load("@build_bazel_rules_apple//apple:swift.bzl", "swift_library")
+
+swift_library(
+    name = "UnitTestsSwiftLib",
+    srcs = glob([
+        "tests/unit/*.swift",
+    ]),
+    deps = [":MotionAnimator"],
+    visibility = ["//visibility:private"],
+)
+
+objc_library(
+    name = "UnitTestsLib",
+    srcs = glob([
+        "tests/unit/*.m",
+    ]),
+    enable_modules = 1,
+    deps = [":MotionAnimator"],
+    visibility = ["//visibility:private"],
+)
+
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
+
+ios_unit_test(
+    name = "UnitTests",
+    deps = [
+      ":UnitTestsLib",
+      ":UnitTestsSwiftLib"
+    ],
+    minimum_os_version = "8.0",
```

Comment:
Why does this unit test application have a minimum OS version? Does the component have a different value for `minimum_os_version`?

Reply:
There isn't a way to specify a minimum OS version for objc_library that I know of.

For a list of examples of usage of this property:
https://github.com/bazelbuild/rules_apple/search?utf8=%E2%9C%93&q=minimum_os_version&type=

Reply:
Yeah, that seems to be the case. https://github.com/bazelbuild/rules_apple/blob/master/apple/ios.bzl

Ack! :D